analytics/analytics.js

```diff
@@ -36,7 +36,7 @@ function decoratedLinkHandler(e) {
     return;
   }
   let anchor = findAnchor(e.target);
-  if (anchor && anchor.hostname != window.location.hostname) {
+  if (anchor && anchor.hostname && anchor.hostname != window.location.hostname) {
```

Comment:
That would also exclude external links, right? Is that intended/expected?

Reply:
It only includes external links. How do you mean they will be excluded?

Reply:
Right, sorry for the noise!